Tan_ShinYi/Assignments/Python_Fundamentals/Names.py

```diff
@@ -0,0 +1,30 @@
+# students = [
+#     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+#     {'first_name' : 'John', 'last_name' : 'Rosales'},
+#     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+#     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+# ]
+#
+# for names in students:
+#     print names['first_name'] + ' ' +names['last_name']
+
+users = {
+    'Students': [
+        {'first_name':  'Michael', 'last_name' : 'Jordan'},
+        {'first_name' : 'John', 'last_name' : 'Rosales'},
+        {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+        {'first_name' : 'KB', 'last_name' : 'Tonel'}
+    ],
+    'Instructors': [
+        {'first_name' : 'Michael', 'last_name' : 'Choi'},
+        {'first_name' : 'Martin', 'last_name' : 'Puryear'}
+    ]
+}
+
+for occupation in users:
+    print occupation
+    count=0
+    for name in users[occupation]:
```

Comment:
or alternatively, you can change line 24 to read `for occupation, user in users:`. and then change this line to read, `for name in user:`